internal/database/sqlite.go

```diff
@@ -78,6 +78,7 @@ func (db *DB) openSQLite() error {
 		return err
 	}
 
+	db.BackupDatabase(false)
```

Comment:
Do we need to run this on both startup and stop?

Reply:
Yeah